package.json

```diff
@@ -6,8 +6,8 @@
   "author": "Symphony",
   "main": "js/main.js",
   "scripts": {
-    "dev": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron .",
-    "demo": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron . --url=file://$(pwd)/demo/index.html",
+    "dev": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron . --enable-sandbox",
```

Comment:
good info, thanks.  above change will effect dev and demo.  also need to figure out how to add for builder.  maybe electron-builder has an option.  i'll look into.